src/includes/classes/MenuPageOptions.php

```diff
@@ -234,10 +234,19 @@ public function __construct()
             echo '      <h3>'.sprintf(__('License Key', SLUG_TD), esc_html(NAME)).'</h3>'."\n";
             echo '      <p><input type="password" name="'.esc_attr(GLOBAL_NS).'[saveOptions][pro_update_password]" value="'.esc_attr($this->plugin->options['pro_update_password']).'" autocomplete="new-password" /></p>'."\n";
 
+            if ((!defined('DISALLOW_FILE_MODS') || !DISALLOW_FILE_MODS) && !apply_filters('automatic_updater_disabled', defined('AUTOMATIC_UPDATER_DISABLED') && AUTOMATIC_UPDATER_DISABLED)) {
```

Comment:
@jaswsinc Shouldn't `&& AUTOMATIC_UPDATER_DISABLED` be `&& !AUTOMATIC_UPDATER_DISABLED`?